wordpress/Makefile

```
@@ -39,23 +51,25 @@ app/build:: .build/wordpress/deployer \
 	mkdir -p "$@"
 
 .build/wordpress/deployer: .build/var/APP_DEPLOYER_IMAGE \
-                           .build/var/MARKETPLACE_TOOLS_TAG \
-                           .build/var/REGISTRY \
-                           .build/var/TAG \
-                           apptest/deployer/* \
-                           apptest/deployer/manifest/* \
-                           deployer/* \
-                           manifest/* \
-                           schema.yaml \
-                           | .build/wordpress
+                                     .build/var/MARKETPLACE_TOOLS_TAG \
+                                     .build/var/REGISTRY \
+                                     .build/var/TRACK \
+                                     .build/var/RELEASE \
+                                     $(shell find apptest/deployer -type f) \
+                                     $(shell find apptest/deployer/manifest -type f) \
```

```suggestion
                                     $(shell find apptest/deployer -type f) \
```

Or we can do a single
$(shell find . -mindepth 2 -type f)

---

Done.

---

Decided to keep `shell find`s for readability, but removed the redundant `$(shell find apptest/deployer/manifest -type f)`